packages/Webkul/Core/src/Helpers/Laravel5Helper.php

```diff
@@ -353,50 +279,66 @@ private function createDownloadableLink(int $productId): void
         ]);
     }
 
-    /**
-     * @param int   $productId
-     * @param array $attributeValues
-     *
-     * @return void
-     */
-    private function createAttributeValues(int $productId, array $attributeValues = []): void
+    private function createAttributeValues(Product $product, array $attributeValues = []): void
     {
         $I = $this;
 
-        $productAttributeValues = [
-            'sku',
-            'url_key',
-            'tax_category_id',
-            'price',
-            'cost',
-            'name',
-            'new',
-            'visible_individually',
-            'featured',
-            'status',
-            'guest_checkout',
-            'short_description',
-            'description',
-            'meta_title',
-            'meta_keywords',
-            'meta_description',
-            'weight',
+        $faker = Factory::create();
+
+        $brand = Attribute::query()
+            ->where(['code' => 'brand'])
+            ->first(); // usually 25
+
+        if (! AttributeOption::query()
+            ->where(['attribute_id' => $brand->id])
+            ->exists()) {
+            AttributeOption::create([
+                'admin_name'   => 'Webkul Demo Brand (c) 2020',
+                'attribute_id' => $brand->id,
+            ]);
+
+        }
+
+        /** @var array $defaultAttributeValues
+         * Some defaults that should apply to all generated products.
+         * By defaults products will be generated as saleable.
+         * If you do not want this, this defaults can be overriden by $attributeValues.
+         */
+        $defaultAttributeValues = [
+            'name'                 => $faker->word,
+            'description'          => $faker->sentence,
+            'short_description'    => $faker->sentence,
+            'sku'                  => $faker->word,
+            'url_key'              => $faker->slug,
+            'status'               => true,
+            'visible_individually' => true,
+            'special_price_from'   => null,
+            'special_price_to'     => null,
+            'special_price'        => null,
+            'price'                => '1.00',
```

Comment:
I would recommend a faked value for 'price'.

Reply:
fixed.